sdk/setup.py

```diff
@@ -22,10 +22,10 @@
 setuptools.setup(
     name="turing-sdk",
     version=version,
-    packages=setuptools.find_packages(),
+    packages=setuptools.find_packages(exclude=["docs", "e2e", "samples", "tests"]),
     install_requires=requirements,
     extras_require={"dev": dev_requirements},
-    python_requires=">=3.7,!=3.10.*",
+    python_requires=">=3.7,!=3.11.*",
```

Comment:
same as the above minor comment

Reply:
Yep, same as above, just updated this. Thanks!